custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -227,7 +227,7 @@ func (t *Translator) metricsFor(values map[string]resource.Quantity, groupResour
 
 	for _, item := range podList.Items {
 		if _, ok := values[fmt.Sprintf("%s", item.GetUID())]; !ok {
-			return nil, provider.NewMetricNotFoundForError(groupResource, metricName, item.GetName())
+			continue
```

Comment:
How about writing some logs in high verbosity level? I think it'll make debugging easier. WDYT?

Reply:
Done.